src/Equinor.ProCoSys.IPO.Command/InvitationCommands/AddComment/AddCommentCommand.cs

```diff
@@ -0,0 +1,19 @@
+﻿using MediatR;
+using ServiceResult;
+
+namespace Equinor.ProCoSys.IPO.Command.InvitationCommands.AddComment
+{
+    public class AddCommentCommand : IRequest<Result<int>>, IInvitationCommandRequest
```

Comment:
Misses tests in AccessValidatorTests.
Each time you create a new command inheriting either IInvitationCommandRequest or IProjectRequest, you should add both negative and positive tests in AccessValidatorTests for particular command

Reply:
.... so I see that these tests are missing for other commands too